audio/audio.c

```diff
@@ -166,6 +167,17 @@ static int audio_open(FAR struct file *filep)
       goto errout_with_lock;
     }
 
+  /* Call open method of lowerhalf if exist */
+
+  if (lower && lower->ops && lower->ops->setup)
+    {
+      ret = lower->ops->setup(lower, tmp);
```

Comment:
cold we call setup only when cref equals 1?

Reply:
Maybe few lower drivers want to know how many users are exists.
So I think this should keep.

Reply:
could you give an example?